copyright

```diff
@@ -260,8 +260,10 @@ Files: images/outfit/tactical?scanner*
 Copyright: Zachary Siple
 License: CC-BY-SA-4.0
 
-Files: images/outfit/korath?rifle*
-Copyright: Evan Fluharty (Evanfluharty@gmai.com)
+Files:
+ images/outfit/korath?rifle*
```

Comment:
just wondering what this line is doing here & not in the merged PR

Reply:
i forgot to put the l at the end of gmail

Reply:
also it was in the PR i just moved it to follow the setup